R/count_labels.R

```diff
@@ -0,0 +1,214 @@
+#' Progress bars with labels
+#'
+#' `shiny` element showing progressbar counts. Each element can have an
+#' unique `id` attribute so each can be used independently.
+#' Progress bar size is dependent on the ratio `choicesnow[i] / countsmax[i]`.
+#' Label is `choices[i] (countsnow[i]/countsmax)`
+#' @param session (`session`) object passed to function given to `shinyServer`.
+#' @param inputId (`character(1)`) `shiny` id
+#' @param choices (`vector`) determines label text.
+#' @param countsmax (`numeric`) determining maximal count of each element.
+#'  Length should be the same as `choices`.
+#' @param countsnow (`numeric`) actual counts of each element.
+#'  Length should be the same as `choices`.
+#' @param counttotal (`numeric(1)`) Determines the size of the whole progress bar.
+#'  For example, in case there are three choices with some counts, one might like
+#'  to display each progress bar with the relative size to the `totalcount`.
+#' @return list of `shiny.tag`
```

Comment:
`counttotal` is an an argument for  `countBars`

Reply:
neither the session

Reply:
session is there - countBars and updateCountBars have a shared documentation page

![image](https://user-images.githubusercontent.com/6959016/220140007-f1f89b33-9683-4030-8e33-a154a4b7177f.png)

Reply:
fair enough,